lib/que/middleware/queue_collector.rb

```diff
@@ -13,28 +13,11 @@ class QueueCollector
         docstring: "Number of jobs in the queue, by job_class/priority/due/failed",
         labels: %i[queue job_class priority due failed],
       )
-      DeadTuples = Prometheus::Client::Gauge.new(
```

Comment:
We also remove the dead tuples, given that's provided to us via our postgres exporter nowadays.